docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md

```diff
@@ -0,0 +1,29 @@
+# [Manual] Notify administrators if several node pools reached its max size
+
+Test verifies that the notification shall be single for all pools if there are several of them reach its max size.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- Perform [2414_1](2414_1.md) case
+
+**Preparations**:
+1. Repeat steps 1-9 of the Preparations of [2414_1](2414_1.md) case 2 times with different docker images (*tool1* and *tool2*)
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Select test *tool1* | |
+| 2 | Repeat steps 7-15 of the [2414_1](2414_1.md) case | |
+| 3 | Repeat steps 1-2 | |
+| 4 | Select test *tool2* | |
+| 5 | Repeat step 2 | | 
+| 6 | Repeat steps 5-6 | |
+| 7 | Open the **Cluster state** page | |
+| 8 | Click the **HOT NODE POOLS** tab | |
+| 9 | Wait until **2/2** nodes is shown in the pool state for both pools created at step 1 of the Preparations |
+| 10 | Wait about 1 minutes | | 
+| 11 | Check admin and non-admin users email | Non-admin and admin users from the Prerequisites receive 1 email for both pools that contains follow information for each pool <li> node pool name <li> node pool maximum size <li> instance type |
+
+**After:**
+- Stop  runs launched at steps 2, 3, 5, 6 from the Preparations
+- Open the **HOT NODE POOLS** tab and delete the pool node created at step 1 of the Preparations
```

Comment:
Fix to `... delete the node pools created ...`

Reply:
Done